tests/parametric/test_config_consistency.py

```diff
@@ -0,0 +1,45 @@
+"""
+Test configuration consistency for features across supported APM SDKs.
+"""
+import pytest
+from utils import scenarios
+
+parametrize = pytest.mark.parametrize
+
+
+def enable_tracing_enabled():
+    env1 = {}
+    env2 = {"DD_TRACE_ENABLED": "true"}
+    return parametrize("library_env", [env1, env2])
+
+
+def enable_tracing_disabled():
+    env = {"DD_TRACE_ENABLED": "false"}
+    return parametrize("library_env", [env])
+
+
+# feature will be added after PR is merged by @zacharycmontaya
+@scenarios.parametric
+class TestTraceEnabled:
```

Comment:
To keep things consistent across tests, let's use the naming schema `Test_Config_<Config-option>_<optional_condition>`

For example, Test_Config_HttpServerErrorStatuses_Default used in Zach's PR ([link](https://github.com/DataDog/system-tests/pull/2980/files#diff-4eba22e107ccb54fd5d8f0acdaf5c8eb131320d4cfbae71c4d664a4616eb1b15))

The <optional-condition> section is critical to weblog (that is, non-parametric) tests because we need different classes to use diff env vars in weblog, but with Parametric, you can use pytests's parameterize.

So in your case, you could keep it under one class and just name it Test_Config_TraceEnabled

Reply:
Also merge w/ main and add @features.tracing_configuration_consistency !